lambda/README.md

```diff
@@ -33,20 +32,32 @@ AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws es \
 --ebs-options EBSEnabled=true,VolumeType=standard,VolumeSize=10
 ```
 
-1. Call the function
+1. Create the SNS Topic
 ```
-AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws lambda \
---endpoint-url=http://localhost:4574 invoke \
---function-name f1 result.log \
---payload '{"Records": [{"EventSource": "foo", "Sns": { "Timestamp": "2014-05-16T08:28:06.801064-04:00", "Message": "Hello world!" }}]}'
+awslocal sns create-topic --name rialto-msg
+```
+
+1. Subscribe the lamba function to the SNS Topic
+```
+awslocal sns subscribe \
```

Comment:
Why `awsloca`l here and `aws` on line 16?

Reply:
I wasn't able to get aws to run properly (will have to check the error) and only got this to run successfully with `awslocal`

Reply:
awslocal is actually just wrapping aws: https://github.com/localstack/awscli-local/blob/master/bin/awslocal#L89